summary_workflows/quantification/APAeval_Q_test_data/data/Qx.mm10.json

```diff
@@ -21,7 +21,8 @@
                 "visualization": {
                     "type": "2D-plot",
                     "x_axis": "Expression_non-matched-PAS_10nt",
-                    "y_axis": "Correlation_coefficient_10nt"
+                    "y_axis": "Correlation_coefficient_10nt",
+                    "optimization": "top-left"
```

Comment:
would it be possible to use the four different optimization values available in this test data so that we can also see what each resulting plot looks like, and then adding the plots in the comments of this PR for our reference?

Reply:
mh well this test data (Qx.mm10) is not the prio, rather the `test_mm10`. 
Also, the main part is the in the template that resides within the docker images, so this should not change. Here, this file is simulating that there is already a result. 
For sure I can manually change the better parameter and show the different results here, would that work?

Reply:
I see! yes that sounds good, it's definitely really helpful to have a reference of how they look like, thank you for doing it! :D I'll approve this PR now since it looks good to me! and the different results you're generating can maybe be added in the description of this PR so it's easy to find

Reply:
Here the results of the 4 nextflow runs with challenge `Qx.mm10` for tool3: `APAeval_Q_test_data/embryonicCortex.PAPERCLIP.mm10.bed`: [tool3_all_corners.tar.gz](https://github.com/iRNA-COSI/APAeval/files/8919915/tool3_all_corners.tar.gz). Reference dataset `Qx.mm10.bed` and genome `genomes/gencode.mm10.annotation.genes.subset.gtf` are used. 
I manually changed the optimization value within `Qx.mm10.json` to the 4 cases: *top-right*, *top-left*, *bottom-left*, *bottom-right*. Then each changed the output name to the corresponding corner and executed `nextlfow run main.nf -profile docker`.
As you can see, the plots display the better parameter with an arrow and the quartiles are computed accordingly. Of course the values don't make sense but this is solely for illustration purposes that the individual corners are used.

Reply:
@dominikburri thank you so much for doing this! I downloaded the folder and looked at the plots from the 4 cases, it's super helpful!! :D

Reply:
yeah sure, many thanks, looks great :D